src/SmartComponents/PatchSetWizard/steps/ReviewPatchSet.js

```diff
@@ -11,6 +11,19 @@ import {
     TextListItemVariants
 } from '@patternfly/react-core';
 import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import { intl } from '../../../Utilities/IntlProvider';
+import messages from '../../../Messages';
+
+const renderTextListItem = (label, text) => (
+    <Fragment>
+        <TextListItem component={TextListItemVariants.dt}>
+            {intl.formatMessage(label)}:
```

Comment:
Maybe it is more clear to pass `label` as a string formatted in advance? I mean, `label` is a message descriptor, whereas `text` is not (it is already a string and was pre-formatted before passing it to this function).